megamek/src/megamek/client/ui/swing/TargetingPhaseDisplay.java

```diff
@@ -1524,15 +1514,21 @@ public void actionPerformed(ActionEvent ev) {
     }
 
     private void updateFlipArms(boolean armsFlipped) {
+        if (ce() == null) {
+            return;
+        }
```

Comment:
This can potentially cause issues... we should really just have a local method variable for the entity, which can then be checked once, or better yet use the entity instead of the id for the whole method.

Reply:
Think you're going to have to elaborate on that. Sometimes, the user doesn't have an entity selected for whatever reason (perhaps they're not the active player), in which case ce() would come back as null. Which would cause an exception on the line immediately following this.

Reply:
Going from memory, the return from ce() might change between calls. So using Entity entity = ce(), then using entity for the method, should prevent issues from that.

Reply:
Theoretically yes. In practice this would mean that the selected entity has been changed or the underlying game object received a "remove entity" update in the middle of this event handler (so, as the user is clicking 'flip arms', they've also somehow managed to simultaneously change the current entity or that entity has become destroyed). It's unlikely to happen during firing/targeting/physical phases, where nothing happens to an entity until the end of a phase. Let's leave it as is for now to keep a lid on scope.